sdk/cosmosdb/cosmos/.npmignore

```diff
@@ -5,4 +5,5 @@ samples
 .gitignore
 *.code-workspace
 *.tgz
-ts-test/
```

Comment:
Can this not be moved up to the root ignore file?

Reply:
I think that the root of the package is the folder that the ```.npmignore``` file is in and it doesn't recurse back up the tree. We can't use ```.gitignore``` (not sure if it recurses up for that either) because strictly speaking we do want to add ```ci.yml``` to the repo.

Another option is to explicitly list the files we want in the package in the ```package.json``` file. This might be safer from a security perspective. I once had a file containing credentials slurped into a package because of this.

Often when I set up ```.npmignore``` files I explicitly exclude everything and then add rules to add things back in (but the ```"files: []``` attribute might be a better choice).

Reply:
I think @KarishmaGhiya has been taking that second approach - explicitly specifying the files we want in the package.json file. I feel like that's a better approach.

Reply:
OK - looking at the packages now I can see most of them have a files attribute. I'll back out my .npmignore changes.